providers/aws/resources/aws_rds.go

```diff
@@ -144,46 +161,11 @@ func (a *mqlAwsRds) getDbInstances(conn *connection.AwsConnection) []*jobpool.Jo
 	return tasks
 }
 
-type mqlAwsRdsDbinstanceInternal struct {
-	cacheSubnets *rdstypes.DBSubnetGroup
-	region       string
-}
-
-func (a *mqlAwsRdsDbinstance) subnets() ([]interface{}, error) {
-	if a.cacheSubnets != nil {
-		res := []interface{}{}
-		conn := a.MqlRuntime.Connection.(*connection.AwsConnection)
-		for i := range a.cacheSubnets.Subnets {
-			subnet := a.cacheSubnets.Subnets[i]
-			sub, err := NewResource(a.MqlRuntime, "aws.vpc.subnet", map[string]*llx.RawData{"arn": llx.StringData(fmt.Sprintf(subnetArnPattern, a.region, conn.AccountId(), convert.ToString(subnet.SubnetIdentifier)))})
-			if err != nil {
-				a.Subnets.State = plugin.StateIsNull | plugin.StateIsSet
-				return nil, err
-			}
-			res = append(res, sub)
-		}
-		return res, nil
-	}
-	return nil, errors.New("no subnets found for RDS DB instance")
-}
-
-func rdsTagsToMap(tags []rdstypes.Tag) map[string]interface{} {
-	tagsMap := make(map[string]interface{})
-
-	if len(tags) > 0 {
-		for i := range tags {
-			tag := tags[i]
-			tagsMap[convert.ToString(tag.Key)] = convert.ToString(tag.Value)
-		}
-	}
-
-	return tagsMap
-}
-
-func (a *mqlAwsRds) dbClusters() ([]interface{}, error) {
+// pendingMaintenanceActions returns all pending maintaince actions for all RDS instances
+func (a *mqlAwsRds) pendingMaintenanceActions() ([]interface{}, error) {
```

Comment:
```suggestion
func (a *mqlAwsRds) allPendingMaintenanceActions() ([]interface{}, error) {
```

if we're fetching all of them?